app/experimenter/nimbus-ui/README.md

```diff
@@ -89,11 +89,35 @@ export interface GetExperimentOverviews {
 
 This app has an [`AppErrorBoundary`](./src/components/AppErrorBoundary/index.tsx) that will capture any uncaught error that occurs and report them to Sentry. This acts as our last line of defense, but ideally we are able to handle errors before they get to this stage. As well, errors that we wish to display to the user should generally be handled in a consistent fashion.
 
-There are two types of errors to account for:
+### General errors
+
+General errors can occur when an operation fails altogether. This could be for any reason, such as when a network request fails, or a bad code path raises an exception. For these errors you'll use React Bootstrap's [`Alert`](https://react-bootstrap.github.io/components/alerts/) components.
+
+For example, if error occurs while saving a record:
+
+```tsx
+<Alert variant="warning">Sorry, there was a problem.</Alert>
+```
+
+_Note:_ displaying technical error messages to the user may not be very helpful, so it's generally advised to either provide a [generic message](./src/lib/constants.ts) or instructions on what to do.
+
+## Form Validation
+
+TODO: Replace storybook links.
```

Comment:
I think you could say "see the form validation section of Storybook for much more detailed info and examples" and the user reading can then check latest on `main` or run locally. (maybe bold this text so it's more obvious?)